iocage/lib/Config/Jail/Properties/Interfaces.py

```diff
@@ -101,25 +72,36 @@ def set(
     def add(
         self,
         jail_if: str,
-        bridges: typing.Optional[typing.Union[str, typing.List[str]]]=None,
-        notify: bool=True
+        bridge_if: typing.Optional[str]=None,
+        notify: typing.Optional[bool]=True
```

Comment:
why is this `Optional`? do we do anything else when it's `None` than when it's `False`?

Reply:
You do not need to specify it - it has a default. So it's okay if you call the function with having notify set to None. That's why the interface says it's optional.

Reply:
python has named parameters
so their position doesn't really matter (except for `self`)
it makes no sense to type a boolean parameter with `Optional` unless the state of not being set is explicitly distinct to the state of being set.